### ISSUE TITLE
grep manual page

### ISSUE THREAD

**Issue by k8hertweck** [datacarpentry/shell-genomics#35]
Teaching a workshop with @ErinBecker and a learner pointed out a cool way to search manuals:

`man cut | grep delimiter`

The example above searches the manual for `cut` for the flag to change the delimiter. 

This would make a great challenge exercise!

**Comment by ErinBecker** [comment]
This sounds like a great idea. Unfortunately it doesn't work. 

`man cut | grep delimiter` gives the output:

> Use delim as the field delimiter character instead of the tab
             delimiter character (see the -d option.)  Output fields are sepa-
             rated by a single occurrence of the field delimiter character.
     -s      Suppress lines with no field delimiter characters.  Unless speci-
             fied, lines with no delimiters are passed through unmodified.

Due to the way the man file is formatted, the word "delimiter" appears on the next line **after** the `-d` flag is introduced:

>     -d delim
       Use delim as the field delimiter character instead of the tab
       character.

This exercise works better if we use the `/` based search option within `man cut`, but that doesn't work as a good example of piping. 

It also doesn't work if we use "delim" as the search term to grep (`man cut | grep "delim"`):

>      terms of column position or in terms of fields delimited by a special
             Use delim as the field delimiter character instead of the tab
             delimiter character (see the -d option.)  Output fields are sepa-
             rated by a single occurrence of the field delimiter character.
     -s      Suppress lines with no field delimiter characters.  Unless speci-
             fied, lines with no delimiters are passed through unmodified.

This works: `man cut | grep -B1 "delimiter"` , but we're getting a little into the weeds here. . . 

So in summary, I think it's a great idea, but it just doesn't play out well in practice. I favor not introducing this new exercise due to the complexity of execution and think it will be more confusing than helpful to learners. I'll leave it to the Maintainers to decide whether to close this issue or incorporate the exercise: @swang8 @smcclatchy @aschuerch

**Comment by swang8** [comment]
The `man cut` output has some specific character, i.e., -^H-d^Hd _^Hd_^He_^Hl_^Hi_^Hm, which makes the grep failed to detect the string "delimiter". It might be good not to use this exercise in the episode.
